book.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] The `:rentals` property here should have a getter and setter, not just a setter. Kindly fix that.